internal/pkg/describe/lb_web_service.go

```diff
@@ -252,6 +265,47 @@ func (e envVars) humanString(w io.Writer) {
 	}
 }
 
+type Secrets struct {
```

Comment:
```suggestion
type Secret struct {
```

Also can this struct be possibly private? I don't think any other pkg from outside will depend on it

Reply:
done